docs/business-apps/power-automate/guidance/working-with-get-items-and-get-files.md

```diff
@@ -12,36 +12,42 @@ The **Get items** and **Get files** SharePoint actions for flows in Power Automa
 **IMPORTANT:** First, and of primary consideration, the **Get items** action only works with lists, whereas the **Get files** action only works with libraries.
 
 ## Item limits
-
-The default item limit is 100.
+The default item limit is 100 and items are paginated by default as well. 
 
 If you are using the default options, and simply specifying the site address and list or library name, Power Automate returns 100 items from the list or library.
 
 If you are working with large lists, you can increase this limit up to the list view threshold limit of 5,000.
 
 To specify the number of items to be returned, expand the **Advanced options**, and specify the number in the **Top Count** action property.
 
+![Modify number of items returned in Get items action](../../../images/flow-get-items-modify-top-count.png)
+
 If you go beyond 5,000 item limit, Power Automate fails and generates an error dialog.
 
-When you select **Apply** to each action to the returned item collection, the items are paginated as well.
+## Limit items to a specific folder
+By default, the action will return all items or files in the list or library, recursively from all folders if available. You can change this behavior with the following:
+* Use _Limit Entries to Folder_ property to select a speific folder in the list or library
```

Comment:
To select a specific folder in the list or library, use _Limit Entries to Folder_ property. (watch for spelling of 'specific.')